CONTRIBUTING.md

```diff
@@ -9,7 +7,68 @@ MongoDB welcomes community contributions! If you’re interested in making a con
+
+The automated workflows require the following secrets to be placed into your Github forks' Settings/Secrets:
+
+```
+AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY
+AWS_REGION
+ATLAS_PUBLIC_KEY
+ATLAS_PRIVATE_KEY
+ATLAS_ORG_ID
+```
+
+## Security - Setup
+
+1. Create and note your [MongoDB Atlas API Key](https://www.mongodb.com/docs/atlas/configure-api-access/#std-label-atlas-admin-api-access).
+2. Create and note your AWS Access Key and AWS Secret Key ID.
+3. Follow the Github docs on how to [create a Secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository) in your clone of this repository.
+
+## Using the MongoDB Atlas CFN Resources
+
+The fastest way to use the resources is with the official [MongoDB Atlas on AWS Partner Solutions](https://aws.amazon.com/solutions/partners/mongodb-atlas/) CloudFormation templates. The [templates](https://github.com/aws-quickstart/quickstart-mongodb-atlas/templates) folder contains concrete CloudFormation templates you can use to start your own projects.
+
+There are two main parts of this project:
+
+1. [quickstart-mongodbatlas](quickstart-mongodbatlas) This is a mirror of https://github.com/aws-quickstart/quickstart-mongodb-atlas.
+
+2. [cfn-resources](cfn-resources) A set of AWS CloudFormation custom resource providers for MongoDB Atlas Resources. Currently, AWS requires users to manually deploy these resources in each AWS region one one desires to use them in. We support this workflow through the standard AWS `cfn submit` tooling. Scripts and Github actions are contained in this repository which demonstrate automating this deployment process.
+
+
+## Registering resources to run locally
+
+1. Please check that you satisfy all the [requirements](#Requirements) before proceeding.
+2. Clone this repo.
+3. A helper script `[cfn-resources/cfn-submit-helper.sh](cfn-submit-helper.sh)` will build and submit each resource for you. You can also run the `cfn submit` tool yourself. Note- this step will soon not be required when AWS launch a public registry.
+
+The following command builds and registers all the types used in the quickstart:
+
+```
+cd mongodbatlas-cloudformation-resources\cfn-resources
+./cfn-submit-helper.sh project cluster database-user project-ip-access-list network-peering
+```
+
+## IAM Access Error When Previsioning Resources
+If you are having difficulty with IAM access, try registering first with the following IAM role [here](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/execute-role.template.yml). This activates the public registry extensions by first using the private registry extensions.
+
+The naming scheme  for a MongoDB Atlas resource on the AWS CloudFormation Third-Party Public Registry is "MongoDB::Atlas::[RESOURCE-NAME]".
```

Comment:
extra space + monospace formatting for naming scheme instead of quotes
```suggestion
The naming scheme for a MongoDB Atlas resource on the AWS CloudFormation Third-Party Public Registry is `MongoDB::Atlas::[RESOURCE-NAME]`.
```